.github/workflows/publish-vectors-pages.yaml

```diff
@@ -0,0 +1,44 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Publish Test Vectors
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
```

Comment:
We don't do either of these for https://github.com/TBD54566975/web5-kt/blob/51ab34faf35ab821a90fae9318f79a3334a4be62/.github/workflows/gh-pages-deploy.yml#L1

Is this something that we should consolidate?

Reply:
Consolidated into Finn's existing GH action